src/Microsoft.ML.Data/DataView/CompositeSchema.cs

```diff
@@ -13,16 +13,16 @@ namespace Microsoft.ML.Data
     /// A convenience class for concatenating several schemas together.
     /// This would be necessary when combining IDataViews through any type of combining operation, for example, zip.
     /// </summary>
```

Comment:
Maybe we need to change this comment, as it seems to be suggesting that zip is not the only use?